### PR TITLE
Fix typos and improve grammar in threading.Barrier docstrings

### PR DESCRIPTION
**Pull request by carlbordum**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Our records indicate we have not received your CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

File: Lib/threading.py
Comment (rougeth) [inline Lib/threading.py:571]: I'd use `unblock` instead of `awoken`. I think there's a typo at beginning of the sentence as well. What do you think of `Threads are blocked on...`?

Comment (carlbordum) [reply Lib/threading.py:571]: I used `awoken` because it is used in the `Barrier.wait` docstring. Would you still prefer `unblock`? And if so, should I change it both places?

I don't see the typo at the beginning of the sentence. English is not my native language, so please show me :)

Comment (rougeth) [reply Lib/threading.py:571]: Since the first part says that the threads were blocked, I think make more sense to use "unblock". For consistency, +1 for changing on `Barrier.wait` too.

I'm not a native speaker as well :) It just seems that `Threads block on 'wait()'...` should be `Threads are blocked on 'wait()'...`.

Comment (carlbordum) [reply Lib/threading.py:571]: To change the terminology in  `threading`, we would have to open a new issue on the tracker. I don't think it has anything to do with this pr. If you think it is an important issue, you can go ahead, I'm not interested.

It will probably end up in a long [red shed](http://red.bikeshed.com/) discussion.

**Review by rougeth — APPROVED** [review]
LGTM

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
Thanks @carlbordum and congrats on your first contribution to CPython :taco:

**Pull request merged**
